kitsune/sumo/static/js/ui.js

```diff
@@ -6,6 +6,17 @@
   $(document).ready(function() {
     initFolding();
     initAnnouncements();
+    $(".delete-profile-btn").removeAttr('disabled');
+
+    $('#delete-profile').submit(function(event){
+      var username = prompt("Are you sure you want to close your account? You cannot undo this action. Please enter your username to proceed.");
+      if(this.username.value == username){
+        this.submit();
+      }else{
+        alert('Sorry the username entered did not match the account. Aborting...');
+        event.preventDefault();
+      }
+    });
```

Comment:
There is a couple of style problems here. 

1. You should use `===` not `==` for comparison.
2. You should put spaces around `{` and `}`. For example, line 15 should be `} else {`, not `}else`{. 
3. Same for the `if` statement. It needs a space after it.

These are little things, but keeping everything consistent is valuable.